Content.Server/_Umbra/Atmos/Components/SpontaneousCombustionComponent.cs

```diff
@@ -0,0 +1,36 @@
+
+namespace Content.Server.Atmos.Components;
```

Comment:
```suggestion
namespace Content.Server._Umbra.Atmos.Components;
```

This shouldn't cause issues and is more accurate.

Reply:
The namespaces were fixed.